js/settings.js

```diff
@@ -100,10 +100,17 @@ export function registerSettings() {
 	game.settings.register(settingsKey, "autoPathfinding", {
 		name: "drag-ruler.settings.autoPathfinding.name",
 		hint: "drag-ruler.settings.autoPathfinding.hint",
-		scpoe: "client",
+		scope: "client",
 		config: early_isGM() || game.settings.get(settingsKey, "allowPathfinding"),
 		type: Boolean,
-		defualt: false,
+		default: false,
+	});
+
+	game.settings.register(settingsKey, "pathfindingRadius", {
```

Comment:
Modifying this setting should wipe the gridless pathfinder cache to reduce unnecessary memory usage. A single cache have a size of multiple Megabytes, so the amount that can be freed could be quite significant.